app/schema.graphql

```diff
@@ -618,6 +618,12 @@ type TraceDatasetInfo {
 
   """Count of total (prompt + completion) tokens in the trace data"""
   tokenCountTotal: Int!
+
+  """Root span latency p50 quantile in milliseconds"""
+  latencyMsP50: Float
+
+  """Root span latency p99 quantile in milliseconds"""
+  latencyMsP99: Float
```

Comment:
nit - Units as a postfix

Reply:
On second thought, i'll keep it a is. Rationale: letting postfix be the part that is "derivative" or "changing", in our case and in our code base overall, the `Ms` part is more "static" than P50, P99, hence the Ps should be postfix instead